network/network_test.go

```diff
@@ -29,7 +30,7 @@ var (
 	testNetworkID        = ""
 )
 
-// Note that this is specific to Joyent Public Cloud and will not pass on
+// Note that this is specific to Triton Public Cloud and will not pass on
```

Comment:
Don't you mean MNX.io?  Do we still have these specifics in MNX.io?

Reply:
This test hasn't been updated since exodus from Joyent. I'm not even sure it's been run since then.

Reply:
Jira ticket is TRITON-2469. I'll prepare a version bump here as well.

Reply:
I'd resolve, but would like @bahamat to make sure as well.

Reply:
I'm not even clear what this test is checking. If it passes in MNX then I'm ok with replacing `Joyent Public Cloud` with `mnx.io's Triton Public Cloud`. If not, then probably just revert it and leave it saying "this only passes on joyent", until we get this test reworked for...whatever.

An even better case would be to change the comment to say "This only works in Triton clouds that are configured with ..." and whatever that criteria is. Any Triton cloud *could* be configured to match what Joyent had (and MNX mostly does these days). So in general it's better to list the specific configuration being tested rather than trying to limit to a single unique cloud that no one else can replicate.

Reply:
The comment is no longer true, in the test output I shared both `TestAccNetworks_List` and `TestAccNetworks_Get` pass. I deleted the comment.